src/pages/waiting-room-to-car/cat-c/waiting-room-to-car.cat-c.page.ts

```diff
@@ -5,12 +5,8 @@ import { Store, select } from '@ngrx/store';
 import { StoreModel } from '../../../shared/models/store.model';
```

Comment:
Are there no tests effected by the deleted functions, or were none of them being tested.

Reply:
Nope, no unit tests affected